### PR TITLE
SGA-90 Change Dropdown designs

### PR DESCRIPTION
**Pull request by ugnspa**
Designed dropdown menu for Goal creation
433563379_826952955918080_1849612743223887501_n
434612690_7798886140162419_4821891110495448797_n

### CODE REVIEW

File: app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt [hunk 1]
```diff
@@ -50,61 +67,68 @@ fun GoalCreation(
             verticalArrangement = Arrangement.Top,
             horizontalAlignment = Alignment.CenterHorizontally,
         ) {
-            Text(text = "Create Goal", style = MaterialTheme.typography.headlineMedium)
+            Text(
+                text = "Create Goal",
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt]: Move text to string resources

File: app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt [hunk 2]
```diff
@@ -50,61 +67,68 @@ fun GoalCreation(
...
+                color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.2f),
+                modifier = Modifier.padding(top = 8.dp, bottom = 8.dp),
+            )
 
-            DropdownMenu(
+            GoalDropdownMenu(
                 options = goalOptions,
                 selectedOption = selectedGoal,
-                label = "Select Goal Period",
+                label = "Period",
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt]: Move text to string resources

File: app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt [hunk 3]
```diff
@@ -50,61 +67,68 @@ fun GoalCreation(
...
             if (selectedGoal.isNotEmpty()) {
                 val exerciseOptions = goalsViewModel.getValidExerciseTypes(selectedGoal)
 
                 if (exerciseOptions.isNotEmpty()) {
-                    DropdownMenu(
+                    ExerciseDropdownMenu(
                         options = exerciseOptions,
                         selectedOption = selectedExercise,
-                        label = "Select Exercise",
+                        label = "Exercise",
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt]: Move text to string resources

File: app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt [hunk 4]
```diff
@@ -50,61 +67,68 @@ fun GoalCreation(
...
                     } else {
                         distanceOptionsWeek
                     }
 
-                    DropdownMenu(
+                    GoalDropdownMenu(
                         options = distanceOptions,
                         selectedOption = selectedDistance,
-                        label = "Select Distance",
+                        label = "Distance",
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt]: Move text to string resources

File: app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt [hunk 5]
```diff
@@ -152,3 +176,184 @@ fun NoValidGoalsMessage() {
...
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                text = label,
+                style = MaterialTheme.typography.bodyLarge,
+            )
+
+            Text(
+                text = selectedOption.ifEmpty { "Select an option" },
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/goals/GoalCreation.kt]: Move text to string resources

**Review by mantasjasikenas — CHANGES_REQUESTED** [review]
Looks good but please move texts to string resources

**Review by mantasjasikenas — APPROVED** [review]

**Review by paulius-mongirdas — APPROVED** [review]

**Pull request merged**